### PR TITLE
Cassandra Computed Tiles release

### PR DESCRIPTION
**Pull request by erikschlegel**
This release is responsible for populating the following cassandra table through Dataframes and Spark SQL

```
CREATE TABLE computedtiles (
    periodstartdate timestamp,
    periodenddate timestamp,
    periodtype text,
    pipelinekey text,
    period text,
    tilez int,
    tilex int,
    tiley int,
    externalsourceid text,
    mentioncount bigint,
    avgsentimentnumerator bigint,
    heatmap text,
    conjunctiontopic1 text,
    conjunctiontopic2 text,
    conjunctiontopic3 text,
    PRIMARY KEY ((periodtype, conjunctiontopic1, conjunctiontopic2, conjunctiontopic3, tilez, period), tilex, tiley, periodstartdate, periodenddate, pipelinekey, externalsourceid)
);
```

### CODE REVIEW

File: project/plugins.sbt
Comment (kevinhartman) [inline project/plugins.sbt]: What's the reason for removing these? Without `sbt-assembly`, we cannot build the project. It may be dependent on some of the above plugins as well.

File: .gitignore
Comment (kevinhartman) [inline .gitignore]: Project directory should **NOT** be excluded.

File: .gitignore
Comment (kevinhartman) [inline .gitignore]: What are the above 2 exclusions guarding against? We haven't had a problem with these in the past. Can we revert this file?

File: out/production/project-fortis-spark/main/resources/ApplicationInsights.xml
Comment (kevinhartman) [inline out/production/project-fortis-spark/main/resources/ApplicationInsights.xml]: We're using environment variables to configure app insights, so we shouldn't need this file.

File: .gitignore
Comment (kevinhartman) [reply .gitignore]: **EDIT**: 
I see the project folder was accidentally excluded. Plz fix :)

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/udfs/FortisUdfFunctions.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/udfs/FortisUdfFunctions.scala]: In the general case, we should not define a usage contract of a method such that a `Double` parameter is expected to be sometimes `null`. Aside, this method seems very specific (a default of 1 when it's called in such a way)—maybe it shouldn't be at this scope? Aside, shouldn't `defaultValue` just be a `Double` that's default value is 1?

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/udfs/FortisUdfFunctions.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/udfs/FortisUdfFunctions.scala]:
```scala
case Some(hm) => hm.mentioncountagg
```

Same fix needed above.

**Review by kevinhartman — CHANGES_REQUESTED** [review]
Looks like some accidental project file edits made it in here. Please revert those before merging :)

File: project/plugins.sbt
Comment (erikschlegel) [reply project/plugins.sbt]: dealing with some issues with my build and didnt realize these were removed until looking at the PR. Hold off on reviewing until I circle back.

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala
Comment (Smarker) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala:13]: spelling: `DetailTileIdColumnName`

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala:16]:
```scala
if (display) column else s"'all' as ${column}"
```

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ComputedTilesAggregator.scala:66]: nit: wrong indent.

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/FortisAggregator.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/FortisAggregator.scala:28]: nite: wrong indent

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/dto/FortisRecords.scala
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/dto/FortisRecords.scala:63]: nit: wrong indent

**Review by kevinhartman — APPROVED** [review]
LGTM, with the caveat that someone who's more familiar with the computed tiles schema & approach should also review (perhaps @jcjimenez).

**Pull request merged**
